kolibri/core/assets/src/constants.js

```diff
@@ -103,6 +103,11 @@ const ERROR_CONSTANTS = {
   NOT_FOUND: 'NOT_FOUND',
   INVALID_CREDENTIALS: 'INVALID_CREDENTIALS',
   MISSING_PASSWORD: 'MISSING_PASSWORD',
+  USER_ALREADY_IN_GROUP_IN_CLASS: 'USER_ALREADY_IN_GROUP_IN_CLASS',
+  MAX_LENGTH: 'MAX_LENGTH',
+  INVALID: 'INVALID',
+  PERMISSION_DENIED: 'PERMISSION_DENIED',
+  NOT_AUTHENTICATED: 'NOT_AUTHENTICATED',
```

Comment:
some new constants are added in this PR, but it's not clear to me why?

It doesn't look like they're thrown by the backend or handled by the frontend

Reply:
I added INVALID, because it is thrown during the validation process, to explicitly handle that. I added the others in order to mirror the list of core python error messages that have been defined, with the expectation that these should be kept in lockstep.

Reply:
Going to merge this PR and submit a clean up PR, as will looking into this more, I found some of the backend constants being referenced in plain text in the frontend, which goes against our usual practices.